tests/sysprep_test.go

```diff
@@ -217,161 +215,120 @@ func insertProductKeyToAnswerFileTemplate(answerFileTemplate string) string {
 	return fmt.Sprintf(answerFileTemplate, productKey)
 }
 
-// addExplicitPodNetworkInterface
-//
-// Deprecated: Use libvmi
-func addExplicitPodNetworkInterface(vmi *v1.VirtualMachineInstance) {
-	vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{*v1.DefaultMasqueradeNetworkInterface()}
-	vmi.Spec.Networks = []v1.Network{*v1.DefaultPodNetwork()}
+func withFeatures(features v1.Features) libvmi.Option {
```

Comment:
Does it make sense to move these options into libvmi?

Reply:
I did not find more usage but we can in follow-up if we find any...

Reply:
I think this is good as it is for private usage.
If we want to move this, in future, to libvmi we should make it less generic (withACPI, withHyperv, etc...)